docs/embed/white-labelling.md

```diff
@@ -0,0 +1,142 @@
+
+- [packages/design-system/src/css/_tokens.scss](https://github.com/n8n-io/n8n/blob/master/packages/design-system/src/css/_tokens.scss)
+- [packages/design-system/src/css/_tokens.dark.scss](https://github.com/n8n-io/n8n/blob/master/packages/design-system/src/css/_tokens.dark.scss)
+
+At the top of `_tokens.scss` you will find `--color-primary` variables as HSL colors:
+
+```scss
+@mixin theme {
+	--color-primary-h: 6.9;
+	--color-primary-s: 100%;
+	--color-primary-l: 67.6%;
+```
+
+In the following example the primary color changed to <span style="color:#0099ff">#0099ff</span>. To convert to HSL you can use a [color converter tool](https://www.w3schools.com/colors/colors_converter.asp).
+
+```scss
+@mixin theme {
+	--color-primary-h: 204;
+	--color-primary-s: 100%;
+	--color-primary-l: 50%;
+```
+
+![Example Theme Color Customization](/_images/embed/white-label/color-transition.gif)
+
+!!! note
+    Similar CSS variables in `_tokens.dark.scss` for Dark mode are an upcoming feature that you can't toggle using n8n's UI yet.
+
+## Theme Logos
+
+To change the editor’s logo assets look into [packages/editor-ui/public](https://github.com/n8n-io/n8n/tree/master/packages/editor-ui/public) and replace:
+
+- favicon-16x16.png
+- favicon-32x32.png
+- favicon.ico
+- n8n-logo.svg
+- n8n-logo-collapsed.svg
+- n8n-logo-expanded.svg
+
+Replace these logo assets used in different Vue.js components like
+
+* [MainSidebar.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/MainSidebar.vue) - top/left logo in the main sidebar
+* [Logo.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/Logo.vue) - Reused in other components
+
+In the following example replace `n8n-logo-collapsed.svg` and `n8n-logo-expanded.svg` to update the main sidebar's logo assets.
+
+![Example Logo Main Sidebar](/_images/embed/white-label/logo-main-sidebar.png)
+
+If your logo assets require different sizing or placement you can customize SCSS styles at the bottom of [MainSidebar.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/MainSidebar.vue).
+
+```scss
+.logoItem {
+	display: flex;
+	justify-content: space-between;
+	height: $header-height;
+	line-height: $header-height;
+	margin: 0 !important;
+	border-radius: 0 !important;
+	border-bottom: var(--border-width-base) var(--border-style-base) var(--color-background-xlight);
+	cursor: default;
+
+	&:hover, &:global(.is-active):hover {
+		background-color: initial !important;
+	}
+
+	* { vertical-align: middle; }
+	.icon {
+		height: 18px;
+		position: relative;
+		left: 6px;
+	}
+
+}
+```
+
+## Text Localization
+
+To change all text occurrences like `n8n` or `n8n.io` to your brand identity you can customize n8n's english internationalization file:
+
+- [packages/editor-ui/src/plugins/i18n/locales/en.json](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/plugins/i18n/locales/en.json)
```

Comment:
delete

Reply:
What do you mean by delete?

Reply:
ah you want to remove the list, gotcha.

Reply:
Yeah remove that line after the previous edit. A single bullet point looks odd.